man/wvtag.1

```diff
@@ -6,7 +6,7 @@
 .Nd manipulate wavpack metadata
 .Sh SYNOPSIS
 .Nm wvtag
-.Op Fl cdhlqvwxy
+.Op Fl options
```

Comment:
I don't see how this is better: we know what the actual option letters are,
and it is customary to list them here, as in e.g. ls(1) or cat(1) or anything else.